PyNE source install error

I'm attempting to install pyne from source following the instructions.
I've pasted the error message I get below.

I presume the pyne version I'm using is the most recent: I git cloned it today. I'm installing on the TACC machine stampede2 (more info here:https://portal.tacc.utexas.edu/user-guides/stampede2).

Unfortunately installing using conda is not an option for me.

Thanks in advance.


```
c455-041[knl](507)$ python setup.py install --user
setup.py:38: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
atomic_data.h generated is the same as existing file, skipping.
atomic_data.cpp 
generated is the same as existing file, skipping.
/opt/apps/cmake/3.16.1/bin/cmake
CMake command is
cmake .. -DCMAKE_INSTALL_PREFIX=/home1/06567/bpthomas/.local `-DPYTHON_EXECUTABLE=/scratch/06567/bpthomas/anaconda3/bin/python`

                                  /   \       
                          )      ((   ))     (                          
(@)                      /|\      ))_((     /|\                          
|-|                     / | \    (/\|/\)   / | \                      (@) 
| | -------------------/--|-voV---\`|'/--Vov-|--\---------------------|-|
|-|                         '^`   (o o)  '^`                          | |
| |                               `\Y/'                               |-|
|-|                                                                   | |
| |        /\             ___           __  __             /\         |-|
|-|       /^~\           / _ \_   _  /\ \ \/__\           /^~\        | |  
| |       /^~\          / /_)/ | | |/  \/ /_\             /^~\        |-|
|-|       /^~\         / ___/| |_| / /\  //__             /^~\        | | 
| |       ^||`         \/     \__, \_\ \/\__/             ^||`        |-|  
|-|        ||                |____/                        ||         | | 
| |       ====                                            ====        |-|
|-|                                                                   | |
| |                                                                   |-|
|-|___________________________________________________________________| |
(@)              l   /\ /         ( (       \ /\   l                `\|-|
                 l /   V           \ \       V   \ l                  (@)
                 l/                _) )_          \I                   
                                   `\ /'
                                     `  


-- Pyne platform defined as: __LINUX__
-- Build type: Release
-- PyNE Fast Compile: TRUE
-- CMAKE_INSTALL_RPATH: /home1/06567/bpthomas/.local/lib://lib
-- CMake Install Prefix: /home1/06567/bpthomas/.local
-- Dependency Root Dir (DEPS_ROOT_DIR): /
-- Dependency Hints (DEPS_HINTS): HINTS;/
-- Dependency Binary Hints (DEPS_BIN_HINTS): HINTS;/;//bin
-- Dependency Library Hints (DEPS_LIB_HINTS): HINTS;/;//lib
-- Dependency Include Hints (DEPS_INCLUDE_HINTS): HINTS;//include
-- HDF5 Root: 
-- HDF5: Using hdf5 compiler wrapper to determine C configuration
--    HDF5 Include directory: /scratch/06567/bpthomas/anaconda3/include
--    HDF5 Library directories: 
--    HDF5 Libraries: /scratch/06567/bpthomas/anaconda3/lib/libhdf5.so;/usr/lib64/librt.so;/usr/lib64/libpthread.so;/scratch/06567/bpthomas/anaconda3/lib/libz.so;/usr/lib64/libdl.so;/usr/lib64/libm.so
-- Found PythonInterp: /scratch/06567/bpthomas/anaconda3/bin/python (found version "3.7.6") 
-- Found PythonLibs: /scratch/06567/bpthomas/anaconda3/lib/libpython3.7m.so (found version "3.7.6") 
-- PYTHON_EXECUTABLE: /scratch/06567/bpthomas/anaconda3/bin/python
-- PYTHON_SITE_PACKAGES: /home1/06567/bpthomas/.local/lib/python3.7/site-packages
-- NUMPY_VERSION_STRING = 1.18.1
-- Found Numpy: 1.18.1  
-- CMAKE_Fortran_IMPLICIT_LINK_DIRECTORIES: /usr/lib/gcc/x86_64-redhat-linux/4.8.5;/usr/lib64;/lib64;/opt/intel/compilers_and_libraries_2018.2.199/linux/tbb/lib/intel64_lin/gcc4.4;/opt/intel/compilers_and_libraries_2018.2.199/linux/daal/lib/intel64_lin;/opt/intel/compilers_and_libraries_2018.2.199/linux/tbb/lib/intel64/gcc4.7;/opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin;/opt/intel/compilers_and_libraries_2018.2.199/linux/compiler/lib/intel64_lin;/opt/intel/compilers_and_libraries_2018.2.199/linux/ipp/lib/intel64;/usr/lib
-- CMAKE_Fortran_IMPLICIT_LINK_LIBRARIES: gfortran;m;gcc_s;gcc;quadmath;m;gcc_s;gcc;c;gcc_s;gcc
-- NUMPY_VERSION_STRING = 1.18.1
-- F2PY Executable: /scratch/06567/bpthomas/anaconda3/bin/f2py
-- F2PY Version: 2
-- A library with LAPACK API found.
-- Found LAPACK Linker Flags: 
-- Found LAPACK Libraries: 
-- Found BLAS Libraries: /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin/libmkl_gf_lp64.so;/opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin/libmkl_gnu_thread.so;/opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin/libmkl_core.so;/opt/apps/gcc/6.3.0/lib64/libgomp.so;-lpthread;-lm;-ldl
-- LIBS: /scratch/06567/bpthomas/anaconda3/lib/libhdf5.so;/usr/lib64/librt.so;/usr/lib64/libpthread.so;/scratch/06567/bpthomas/anaconda3/lib/libz.so;/usr/lib64/libdl.so;/usr/lib64/libm.so;/scratch/06567/bpthomas/anaconda3/lib/libpython3.7m.so
-- decay.cpp exists, no need to download or extract!
-- decay-linux-gnu.s exists, no need to download or extract!
-- cram.c exists, no need to download or extract!
-- cram-linux-gnu.s exists, no need to download or extract!
-- Compiling decay.cpp fast from assembly decay-linux-gnu.s
-- Compiling cram.c fast from assembly cram-linux-gnu.s
-- Include paths for /scratch/06567/bpthomas/pyne/src: /scratch/06567/bpthomas/anaconda3/include;/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include;/scratch/06567/bpthomas/pyne/build/pyne;/scratch/06567/bpthomas/pyne/src
-- Include paths for /scratch/06567/bpthomas/pyne/pyne: /scratch/06567/bpthomas/anaconda3/include;/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include;/scratch/06567/bpthomas/pyne/build/pyne;/scratch/06567/bpthomas/pyne/src
-- Modules to Cythonize: ace;bins;data;dtypes;endf;enrichment;extra_types;jsoncpp;material;nucname;particle;pyne_config;rxname;source;stlcontainers;tally;transmuters;_utils
-- F2PY F90 FLAGS: 
-- Include paths for /scratch/06567/bpthomas/pyne/pyne/xs: /scratch/06567/bpthomas/anaconda3/include;/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include;/scratch/06567/bpthomas/pyne/build/pyne;/scratch/06567/bpthomas/pyne/src
-- Configuring done
CMake Warning at src/CMakeLists.txt:87 (add_library):
  Cannot generate a safe runtime search path for target pyne because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target material because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target cram because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:46 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target extra_types because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target bins because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target tally because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target endf because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target dtypes because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target jsoncpp because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target data because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target nucname because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target rxname because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target enrichment because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target source because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target ace because files in
  some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target pyne_config because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target stlcontainers because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target transmuters because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target particle because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target _utils because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/CMakeLists.txt:38 (cython_add_module)


CMake Warning at /opt/apps/cmake/3.16.1/share/cmake-3.16/Modules/FindPythonLibs.cmake:337 (add_library):
  Cannot generate a safe runtime search path for target models because files
  in some directories may conflict with libraries in implicit directories:

    runtime library [libmkl_gf_lp64.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_gnu_thread.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib
    runtime library [libmkl_core.so] in /opt/intel/compilers_and_libraries_2018.2.199/linux/mkl/lib/intel64_lin may be hidden by files in:
      /scratch/06567/bpthomas/anaconda3/lib

  Some of these libraries may not be found correctly.
Call Stack (most recent call first):
  cmake/UseCython.cmake:273 (python_add_module)
  pyne/xs/CMakeLists.txt:9 (cython_add_module)


-- Generating done
-- Build files have been written to: /scratch/06567/bpthomas/pyne/build
[  0%] Built target ReplicatePythonSourceTree
[  0%] Building CXX object src/CMakeFiles/pyne.dir/utils.cpp.o
/scratch/06567/bpthomas/pyne/src/utils.cpp: In function ‘std::ostringstream pyne::comment_line_wrapping(std::string, std::string, int)’:
/scratch/06567/bpthomas/pyne/src/utils.cpp:184:10: error: use of deleted function ‘std::basic_ostringstream<char>::basic_ostringstream(const std::basic_ostringstream<char>&)’
   return oss;
          ^
In file included from /scratch/06567/bpthomas/pyne/src/utils.h:15:0,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/sstream:387:11: note: ‘std::basic_ostringstream<char>::basic_ostringstream(const std::basic_ostringstream<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_ostringstream : public basic_ostream<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/sstream:387:11: error: use of deleted function ‘std::basic_ostream<char>::basic_ostream(const std::basic_ostream<char>&)’
In file included from /usr/include/c++/4.8.2/istream:39:0,
                 from /usr/include/c++/4.8.2/sstream:38,
                 from /scratch/06567/bpthomas/pyne/src/utils.h:15,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/ostream:58:11: note: ‘std::basic_ostream<char>::basic_ostream(const std::basic_ostream<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_ostream : virtual public basic_ios<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/ostream:58:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
In file included from /usr/include/c++/4.8.2/ios:44:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/sstream:38,
                 from /scratch/06567/bpthomas/pyne/src/utils.h:15,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/bits/basic_ios.h:66:11: note: ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_ios : public ios_base
           ^
In file included from /usr/include/c++/4.8.2/iomanip:40:0,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:5:
/usr/include/c++/4.8.2/bits/ios_base.h:786:5: error: ‘std::ios_base::ios_base(const std::ios_base&)’ is private
     ios_base(const ios_base&);
     ^
In file included from /usr/include/c++/4.8.2/ios:44:0,
                 from /usr/include/c++/4.8.2/istream:38,
                 from /usr/include/c++/4.8.2/sstream:38,
                 from /scratch/06567/bpthomas/pyne/src/utils.h:15,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/bits/basic_ios.h:66:11: error: within this context
     class basic_ios : public ios_base
           ^
In file included from /scratch/06567/bpthomas/pyne/src/utils.h:15:0,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/sstream:387:11: error: use of deleted function ‘std::basic_ios<char>::basic_ios(const std::basic_ios<char>&)’
     class basic_ostringstream : public basic_ostream<_CharT, _Traits>
           ^
/usr/include/c++/4.8.2/sstream:387:11: error: use of deleted function ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’
/usr/include/c++/4.8.2/sstream:64:11: note: ‘std::basic_stringbuf<char>::basic_stringbuf(const std::basic_stringbuf<char>&)’ is implicitly deleted because the default definition would be ill-formed:
     class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
           ^
In file included from /usr/include/c++/4.8.2/bits/locale_facets.h:44:0,
                 from /usr/include/c++/4.8.2/locale:40,
                 from /usr/include/c++/4.8.2/iomanip:43,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:5:
/usr/include/c++/4.8.2/streambuf:802:7: error: ‘std::basic_streambuf<_CharT, _Traits>::basic_streambuf(const std::basic_streambuf<_CharT, _Traits>&) [with _CharT = char; _Traits = std::char_traits<char>]’ is private
       basic_streambuf(const basic_streambuf& __sb)
       ^
In file included from /scratch/06567/bpthomas/pyne/src/utils.h:15:0,
                 from /scratch/06567/bpthomas/pyne/src/utils.cpp:8:
/usr/include/c++/4.8.2/sstream:64:11: error: within this context
     class basic_stringbuf : public basic_streambuf<_CharT, _Traits>
           ^
make[2]: *** [src/CMakeFiles/pyne.dir/utils.cpp.o] Error 1
make[1]: *** [src/CMakeFiles/pyne.dir/all] Error 2
make: *** [all] Error 2

--------------------

If you are having issues building pyne, please report your problem to pyne-dev@googlegroups.com or look for help at http://pyne.io

--------------------
Traceback (most recent call last):
  File "setup.p### y", line 425, in <module>
    main()
  File "setup.py", line 409, in main
    main_safe(ns)
  File "setup.py", line 401, in main_safe
    main_body(ns)
  File "setup.py", line 384, in main_body
    rtn = subprocess.check_call(['make'] + ns.make_args, cwd=ns.build_dir)
  File "/scratch/06567/bpthomas/anaconda3/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make']' returned non-zero exit status 2.

Thanks, that seems to have solved the original problem.

The install now gets almost to the end before it hits a bug, apparently to do with fortran compiler versions. 
Any advice here? I include didn't include the whole error trace, but this should be the pertinent part:

```[ 97%] Linking CXX shared module _utils.so
[ 97%] Built target _utils
Scanning dependencies of target transport_spatial_methods
[ 98%] Generating transport_spatial_methods_so
running build
running config_cc
unifing config_cc, config, build_clib, build_ext, build commands --compiler options
running config_fc
unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
running build_src
build_src
building extension "transport_spatial_methods" sources
f2py options: ['--no-wrap-functions']
f2py:> /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methodsmodule.c
creating /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7
Reading fortran codes...
	Reading file '/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90' (format:free)
Post-processing...
	Block: transport_spatial_methods
			Block: main
In: :transport_spatial_methods:/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90:main
get_useparameters: no module invar info used by main
In: :transport_spatial_methods:/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90:main
get_useparameters: no module solvar info used by main
In: :transport_spatial_methods:/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90:main
get_useparameters: no module timevar info used by main
In: :transport_spatial_methods:/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90:main
get_useparameters: no module precision_module info used by main
Post-processing (stage 2)...
Building modules...
	Building module "transport_spatial_methods"...
		Creating wrapper for Fortran subroutine "main"("main")...
		Constructing wrapper function "main"...
		  qdfile,xsfile,srcfile,mtfile,inflow_file,phi_file,fluxout,error_code_out,tsolve_out,ttosolve_out,tend_out = main(titlein,solver_in,solver_type_in,spatial_order_in,angular_quadrature_order_in,qdtypin,nodes_x_in,nodes_y_in,nodes_z_in,num_groups_in,num_materials_in,x_cell_widths_in,y_cell_widths_in,z_cell_widths_in,x_boundry_condition_1_in,x_boundry_condition_2_in,y_boundry_condition_1_in,y_boundry_condition_2_in,z_boundry_condition_1_in,z_boundry_condition_2_in,material_id_in,quadrature_file,xs_file_in,source_input_file_in,bc_input_filein,flux_output_filein,convergence_criterion_in,itmxin,moments_converged_in,converge_tolerence_in,max_mom_printed_in,moment_sum_flag_in,mom_at_a_pt_flag_in,quad_flux_print_flag_in)
	Wrote C/API module "transport_spatial_methods" to file "/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methodsmodule.c"
	Fortran 77 wrappers are saved to "/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methods-f2pywrappers.f"
  adding '/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/fortranobject.c' to sources.
  adding '/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7' to include_dirs.
copying /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/f2py/src/fortranobject.c -> /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7
copying /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/f2py/src/fortranobject.h -> /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7
  adding '/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methods-f2pywrappers.f' to sources.
build_src: building npy-pkg config files
running build_ext
customize UnixCCompiler
customize UnixCCompiler using build_ext
get_default_fcompiler: matching types: '['gnu95', 'intel', 'lahey', 'pg', 'absoft', 'nag', 'vast', 'compaq', 'intele', 'intelem', 'gnu', 'g95', 'pathf95', 'nagfor']'
customize Gnu95FCompiler
Found executable /opt/apps/gcc/6.3.0/bin/gfortran
customize Gnu95FCompiler
customize Gnu95FCompiler using build_ext
building 'transport_spatial_methods' extension
compiling C sources
C compiler: gcc -pthread -B /scratch/06567/bpthomas/anaconda3/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC

creating /tmp/tmpxkqo8xrk/tmp
creating /tmp/tmpxkqo8xrk/tmp/tmpxkqo8xrk
creating /tmp/tmpxkqo8xrk/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7
compile options: '-I/scratch/06567/bpthomas/pyne-gonuke/pyne/build/src -I/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7 -I/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include -I/scratch/06567/bpthomas/anaconda3/include/python3.7m -c'
gcc: /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/fortranobject.c
gcc: /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methodsmodule.c
In file included from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832:0,
                 from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/fortranobject.h:13,
                 from /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/fortranobject.c:2:
/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^~~~~~~
In file included from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832:0,
                 from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/fortranobject.h:13,
                 from /tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methodsmodule.c:16:
/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^~~~~~~
/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7/transport_spatial_methodsmodule.c:170:12: warning: ‘f2py_size’ defined but not used [-Wunused-function]
 static int f2py_size(PyArrayObject* var, ...)
            ^~~~~~~~~
compiling Fortran sources
Fortran f77 compiler: /opt/apps/gcc/6.3.0/bin/gfortran -Wall -g -ffixed-form -fno-second-underscore -fPIC -O3 -funroll-loops
Fortran f90 compiler: /opt/apps/gcc/6.3.0/bin/gfortran -fdefault-real-8 -fPIC -O3 -funroll-loops
Fortran fix compiler: /opt/apps/gcc/6.3.0/bin/gfortran -Wall -g -ffixed-form -fno-second-underscore -fdefault-real-8 -fPIC -O3 -funroll-loops
creating /tmp/tmpxkqo8xrk/scratch
creating /tmp/tmpxkqo8xrk/scratch/06567
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke/pyne
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke/pyne/src
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods
creating /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d
compile options: '-I/scratch/06567/bpthomas/pyne-gonuke/pyne/build/src -I/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7 -I/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include -I/scratch/06567/bpthomas/anaconda3/include/python3.7m -c'
gfortran:f90: /scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90
/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90:98:4:

 USE invar
    1
Fatal Error: Cannot read module file ‘invar.mod’ opened at (1), because it was created by a different version of GNU Fortran
compilation terminated.
error: Command "/opt/apps/gcc/6.3.0/bin/gfortran -fdefault-real-8 -fPIC -O3 -funroll-loops -I/scratch/06567/bpthomas/pyne-gonuke/pyne/build/src -I/tmp/tmpxkqo8xrk/src.linux-x86_64-3.7 -I/scratch/06567/bpthomas/anaconda3/lib/python3.7/site-packages/numpy/core/include -I/scratch/06567/bpthomas/anaconda3/include/python3.7m -c -c /scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.f90 -o /tmp/tmpxkqo8xrk/scratch/06567/bpthomas/pyne-gonuke/pyne/src/transport_spatial_methods/3d/main.o" failed with exit status 1
make[2]: *** [pyne/transport_spatial_methods_so] Error 1
make[1]: *** [pyne/CMakeFiles/transport_spatial_methods.dir/all] Error 2
make: *** [all] Error 2

--------------------

If you are having issues building pyne, please report your problem to pyne-dev@googlegroups.com or look for help at http://pyne.io

--------------------
Traceback (most recent call last):
  File "setup.py", line 409, in <module>
    main()
  File "setup.py", line 393, in main
    main_safe(ns)
  File "setup.py", line 385, in main_safe
    main_body(ns)
  File "setup.py", line 368, in main_body
    rtn = subprocess.check_call(['make'] + ns.make_args, cwd=ns.build_dir)
  File "/scratch/06567/bpthomas/anaconda3/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make']' returned non-zero exit status 2.

--- a/news/fix_ostringstream_compiler_issue.rst
+++ b/news/fix_ostringstream_compiler_issue.rst
@@ -1,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:** 
+
+* change return type of method to avoid compiler compatibility issue
+
+**Security:** None

--- a/src/material.cpp
+++ b/src/material.cpp
@@ -847,7 +847,7 @@ std::string pyne::Material::mcnp(std::string frac_type, bool mult_den) {
   // Metadata comments
   if (metadata.isMember("comments")) {
     std::string comment_string = "comments: " + metadata["comments"].asString();
-    oss << pyne::comment_line_wrapping(comment_string, comment_prefix, mcnp_line_length).str();
+    oss << pyne::comment_line_wrapping(comment_string, comment_prefix, mcnp_line_length);
   }
 
   // Metadata mat_num
@@ -884,7 +884,7 @@ std::string pyne::Material::phits(std::string frac_type, bool mult_den) {
   // Metadata comments
   if (metadata.isMember("comments")) {
     std::string comment_string = "comments: " + metadata["comments"].asString();
-    oss << pyne::comment_line_wrapping(comment_string, comment_prefix, mcnp_line_length).str();
+    oss << pyne::comment_line_wrapping(comment_string, comment_prefix, mcnp_line_length);
   }
 
   // Metadata mat_num

--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -164,7 +164,7 @@ std::string pyne::to_lower(std::string s) {
   return s;
 }
 
-std::ostringstream pyne::comment_line_wrapping(std::string line,
+std::string pyne::comment_line_wrapping(std::string line,
                                                std::string comment_prefix,
                                                int line_length) {
   std::ostringstream oss;
@@ -181,7 +181,7 @@ std::ostringstream pyne::comment_line_wrapping(std::string line,
     oss << comment_prefix << line << std::endl;
   }
 
-  return oss;
+  return oss.str();
 }
 
 std::string pyne::capitalize(std::string s) {

--- a/src/utils.h
+++ b/src/utils.h
@@ -90,7 +90,7 @@ namespace pyne {
   std::string capitalize(std::string s);
 
   /// Forms and returns the wrapped lines with a lenght up to line_lenght.
-  std::ostringstream comment_line_wrapping(std::string line, std::string comment_prefix = "",
+  std::string comment_line_wrapping(std::string line, std::string comment_prefix = "",
                                            int line_length = 79);
 
   /// Finds and returns the first white-space delimited token of a line.

